larana/OpticalDetector/SimPhotonCounter_module.cc

```diff
@@ -430,15 +440,16 @@ namespace opdet {
                 //Get arrival time from phot
                 fTime= Phot.Time;
 
-		// special case for LibraryBuildJob: no working "Reflected" handle and all photons stored in single object - must sort using wavelength instead
-		if(pvs->IsBuildJob() && !Reflected) { // all photons contained in object with Reflected = false flag
-	 	  // Increment per OpDet counters and fill per phot trees
+                // special case for LibraryBuildJob: no working "Reflected" handle and all photons stored in single object - must sort using wavelength instead
+                if(fPVS->IsBuildJob() && !Reflected) {
+                  // all photons contained in object with Reflected = false flag
+                  // Increment per OpDet counters and fill per phot trees
                   fCountOpDetAll++;
                   if(fMakeAllPhotonsTree){
-		    if (fWavelength < 200 || (pvs->StoreReflected() && fWavelength > 200)) {
-         	      fThePhotonTreeAll->Fill();
+                    if (fWavelength < 200 || (fPVS->StoreReflected() && fWavelength > 200)) {
```

Comment:
Really minor point but could these hardcoded numbers be set as const data members at the start of the file as it would make maintenance easier?

Reply:
Also commit 27c8c1e5.